File: StudentSystem.Models/Specialty.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace StudentSystemDbContext.Models
{
    public class Specialty
    {
        public int Id { get; set; }

        public String Name { get; set; }

        public Faculty Faculty { get; set; }

        public IEnumerable<Student> Students { get; set; }

        public IEnumerable<Lecturer> Lecturers { get; set; }
    }
}

File: db/models/Specialty.js

module.exports = function (sequelize, DataTypes) {
    const Sale = sequelize.define('Specialty', {
        id: {
            type: DataTypes.INTEGER,
            autoIncrement: true,
            primaryKey: true
        },
        name: {
            type: DataTypes.STRING
        },
        faculty: {
            type: DataTypes.INTEGER
        },
        students: {
            type: DataTypes.STRING
        },
        lecturers: {
            type: DataTypes.INTEGER
        }
    });
    
    // Class Method
    Sale.associate = function (models) {
    };
    
    // Instance Method
    // Sale.prototype.someMethod = function () {
    // };

    return Sale;
};